api/src/main/java/org/openmrs/api/impl/UserServiceImpl.java

```diff
@@ -628,8 +628,26 @@ public void changePassword(User user, String oldPassword, String newPassword) th
 			throw new APIException("old.password.not.correct", (Object[]) null);
 		}
 	
+		updatePassword(user, newPassword);
+	}
+
+	private void updatePassword(User user, String newPassword) {
```

Comment:
I thought you were going to update the other changePassword method too to delegate to this new utility method